epochX/cudacpp/CODEGEN/PLUGIN/CUDACPP_SA_OUTPUT/patchMad.sh

```diff
@@ -40,51 +40,59 @@ dir_patches=$2
 
 if [ ! -e ${dir} ]; then echo "ERROR! Directory $dir does not exist"; exit 1; fi
 
-# These two steps are part of "cd Source; make" but they actually are code-generating steps
-#${dir}/bin/madevent treatcards run
-#${dir}/bin/madevent treatcards param
-
-# Cleanup
-#\rm -f ${dir}/crossx.html
-#\rm -f ${dir}/index.html
-#\rm -f ${dir}/madevent.tar.gz
-#\rm -f ${dir}/Cards/delphes_trigger.dat
-#\rm -f ${dir}/Cards/plot_card.dat
-#\rm -f ${dir}/bin/internal/run_plot*
-#\rm -f ${dir}/HTML/*
-#\rm -rf ${dir}/bin/internal/__pycache__
-#\rm -rf ${dir}/bin/internal/ufomodel/__pycache__
-#touch ${dir}/HTML/.keep # new file
+# AV Recover special 'tmad' mode used by generateAndCompare.sh, after OM's changes that commented this out in patchMad.sh
```

Comment:
Would not make more sense to have all those lines in the generateandcompare.sh script? (But no strong feeling) but maybe add a comment here that in "normal" mode tmadmode is zero